backend/internal/controller/server/msg_handler.go

```diff
@@ -66,3 +67,26 @@ func (s server) createResponse(reader io.Reader) ([]byte, error) {
 func (s server) ResetScoreHandler(w http.ResponseWriter, r *http.Request) {
 	s.game.ResetScore()
 }
+
+//Incoming URL should be in the format:
+// '/goal?action=[add/sub]&team=[1/2]'
+// team 1 - White; team 2 - Blue
+func (s server) ManipulateScoreHandler(w http.ResponseWriter, r *http.Request) {
+
+	team := r.URL.Query().Get("team")
+
+	teamID, err := strconv.Atoi(team)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	switch action := r.URL.Query().Get("action"); action {
+	case "add":
+		s.game.AddGoal(teamID)
+	case "sub":
+		// PLACEHOLDER
+		//s.game.SubGoal(teamID)
+	default:
+		log.Panic("Wrong action")
```

Comment:
This should return `400 Bad Request` and shouldn't call `panic` - it's a handled error, no need to panic ;)